examples/basic/cli/src/OntologyType.ts

```diff
@@ -17,12 +17,6 @@
 import type { OsdkObjectFrom } from "@osdk/client";
 import type { Ontology } from "@osdk/examples.basic.sdk";
 
-// This little trick makes it easier for me to see
-// what is happening in the types when I hover.
-// We need to actually generate some nice types here too
-type CapturedOntologyType = typeof Ontology;
-export interface OntologyType extends CapturedOntologyType {}
-
 // Demo for if you want concrete types
-export interface Employee extends OsdkObjectFrom<"Employee", OntologyType> {}
-export interface Todo extends OsdkObjectFrom<"Todo", OntologyType> {}
+export interface Employee extends OsdkObjectFrom<"Employee", Ontology> {}
```

Comment:
Now that the Ontology file does the capture and type export this wasnt needed and confused people